main.py

```diff
@@ -75,11 +102,12 @@ def compare_settings(args):
 		for k in changed_keys:
 			print(f'{k}: {previous_args_dict.get(k)} --> {current_args_dict.get(k)}')
 
+
 def profiling(args):
-	import cProfile, pstats
+	import cProfile
```

Comment:
`import pstats` is useful when profiling, it means that sample code below without needing additional imports. Not mandatory but more "confortable"